src/cmd/health.js

```diff
@@ -425,7 +425,6 @@ async function main(argv = minimist(process.argv.slice(2))) {
             const health = new NSFSHealth({ https_port, config_root, all_account_details, all_bucket_details, check_syslog_ng });
             const health_status = await health.nc_nsfs_health();
             process.stdout.write(JSON.stringify(health_status) + '\n');
```

Comment:
@naveenpaul1 Can you try this approach using the write callback to exit only after flushing the output?
```suggestion
            process.stdout.write(JSON.stringify(health_status) + '\n', () => {
                process.exit(0);
            });
```

Reply:
Sure @guymguym

Reply:
did one round of testing with monitoring team and this fix is working